content/sensu-go/5.16/reference/backend.md

```diff
@@ -605,6 +609,34 @@ sensu-backend start --etcd-client-cert-auth
 etcd-client-cert-auth: true{{< /highlight >}}
 
 
+| etcd-discovery        |      |
+------------------------|------
+description             | Exposes [etcd's embedded auto-discovery features][17]. Attempts to use [etcd discovery][18] to get the cluster configuration.
+type                    | String
+default                 | ""
+example                 | {{< highlight shell >}}# Command line example
+sensu-backend start --etcd-discovery https://10.0.0.1:2380
+
+# /etc/sensu/backend.yml example
+etcd-discovery:
+  - https://10.0.0.1:2380
+{{< /highlight >}}
+
+
+| etcd-discovery-srv    |      |
+------------------------|------
+description             | Exposes [etcd's embedded auto-discovery features][17]. Attempts to use a [DNS SRV][19] record to get the cluster configuration.
+type                    | String
+default                 | ""
+example                 | {{< highlight shell >}}# Command line example
+sensu-backend start --etcd-discovery-srv https://10.0.0.1:2380
```

Comment:
In the case of DNS SRV, the argument is simply a domain name, such as `example.org` or `test.com`. Etcd then uses the [SRV record](https://en.wikipedia.org/wiki/SRV_record) of that domain name to discover the cluster configuration.